pkg/server/updates.go

```diff
@@ -183,6 +184,152 @@ func addInfoToURL(ctx context.Context, url *url.URL, s *Server, n diagnosticspb.
 	url.RawQuery = q.Encode()
 }
 
+const (
+	aws   = "Amazon Web Services"
```

Comment:
pkg/server is a pretty hefty package -- given that all this is pretty standalone logic, I might suggest pulling everything except the call into a new package like `pkg/util/couldinfo` or something?